.all-contributorsrc

```diff
@@ -1218,7 +1218,7 @@
       "login": "moradabaz",
       "name": "Morad :)",
       "avatar_url": "https://avatars.githubusercontent.com/u/29915156?v=4",
-      "profile": "https://moradabaz.github.io/",
+      "profile": "https://moradisten.github.io/",
```

Comment:
why are you modifying this, btw?

Reply:
Not intentionally, it got slipped through while merging main branch into the forked one. I'll update that.